ai-backend/config/agent_config.py

```
@@ -4,7 +4,8 @@
 Configuration settings for AI agents focused on football (soccer) journalism.
 """
 
-from typing import Dict, Any
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)